cmd/achcli/main.go

```diff
@@ -29,6 +30,8 @@ var (
 
 	flagPretty        = flag.Bool("pretty", false, "Display all values in their human readable format")
 	flagPrettyAmounts = flag.Bool("pretty.amounts", false, "Display human readable amounts instead of exact values")
+
+	flagConfigFileName = flag.String("allow-validate-opts", "", "Path to config file in json format to enable validation opts")
```

Comment:
This name is too long. Maybe `-validate <filepath>`? We could allow `-validate.requireABAOrigin false` by reading the struct.